Resources/Prototypes/Entities/Objects/Weapons/Guns/Battery/battery_guns.yml

```diff
@@ -540,15 +540,19 @@
         shader: unshaded
   - type: Gun
     selectedMode: FullAuto
-    fireRate: 4.5
+    angleDecay: 3 # DeltaV
+    angleIncrease: 2 # DeltaV
+    fireRate: 8 # DeltaV - was 4.5
+    minAngle: 4 # DeltaV
+    maxAngle: 18 # DeltaV
```

Comment:
just have begin end comments for the 4 added fields and dont put firerate in the middle of them